make.ps1

```diff
@@ -122,13 +122,20 @@ function Check-Command
 	{
 		echo "Checking for explicit interface violations..."
 		Invoke-Expression "$utilityPath $modID --check-explicit-interfaces"
+	}
+	else
+	{
+		UtilityNotFound
+	}
 
+	if (Test-Path "$env:ENGINE_DIRECTORY/OpenRA.StyleCheck.exe")
```

Comment:
For consistency can you define a $styleCheckPath instead of hardcoding this x 3?